doc/ImplementingApplications/ApplicationPatternDocumentation/ApplicationPatternDocumentation.md

```diff
@@ -336,7 +336,7 @@ This class provides functionality to create a HTTP response header.
 |---|---|---|
 |**xCorrelatorGenerator** <br>This function generates a xCorrelator based on the regular expression provided in the specification.||{promise} return the xCorrelator.| 
 |**executionTimeInMilliseconds** <br>This function calculates the execution time for processing the request.|{JSONObject} **startTime** start time of the request. |{promise} return the execution time in milli seconds.| 
-|**createResponseHeader** <br>This function creates response header based on the provided input values. |{JSONObject} **xCorrelator** of the request. <br>{JSONObject} **startTime** start time of the request. <br>{JSONObject} **operationName** of the request.|{promise} return the response header.| 
+|**createResponseHeader** <br>This function creates response header based on the provided input values. |{JSONObject} **xCorrelator** of the request. <br>{JSONObject} **startTime** start time of the request. <br>{JSONObject} **operationName** of the request.<br>{JSONObject} **backendTime** of the request.|{promise} return the response header.| 
```

Comment:
I removed the changes in documentation in this PR and made them in PR #617 (otherwise there would be a conflict) . IMHO manually creating such documentation is not feasible long-term.